### PR TITLE
Adding golang

### PR DESCRIPTION
**Pull request by fabianmet**
Adds section about Golang, huge contribution to the OSS world.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
+    "notables": [
+      "Displaying over 100 million CAPTCHAs every day",
+      "Google pivoted the service away from its original concept, with a focus on reducing the amount of user interaction needed to verify a user"
+    ]
+  },
+  {
+    "title": "Golang",
+    "description": "The Golang or Go language. Many of the tools used in the OSS world are written in Go and it is a great language to get started in.",
+    "value": "Even though there are some critisms about what Go does or does not do, it is a very opinionated language with a very powerfull standard library. Many open source tools used every day across the globe are written in Go, and it is the ease of distribution and packaging that makes it that not everyone realises they are already using Golang projects.",
+    "img": "img/recaptcha.png",
```
Comment (wiardvanrij) [inline data.json]: This still links to the wrong image ;)

**Review by wiardvanrij — APPROVED** [review]

**Review by fabianmet — COMMENTED** [review]
Formatting and other img

**Comment by wiardvanrij** [comment]
@all-contributors please add @fabianmet for adding an entry

**Comment by wiardvanrij** [comment]
@all-contributors please add @fabianmet for Content

**Comment by wiardvanrij** [comment]
@all-contributors please add @fabianmet for Content

**Pull request merged**